projects/kit/components/input-number/input-number.component.ts

```diff
@@ -188,6 +190,7 @@ export class TuiInputNumberComponent
                 this.numberFormat.thousandSeparator,
                 nativeFocusableElement,
                 allowNegative,
+                this.isIOS,
```

Comment:
please, check the correctness of work `input-slider.ts` on demo

Reply:
behavior incorrect, but same behavior in main demo. I think this is a separate bug

Reply:
could you fix it in this PR or would you like it in a separate one?